src/appshell/qml/platform/PlatformMenuBar.qml

```diff
@@ -100,6 +100,7 @@ Item {
         menuItem.checkable = itemInfo.checkable
         menuItem.separator = !Boolean(itemInfo.title)
         menuItem.role = itemInfo.role
+        menuItem.icon.mask = true
```

Comment:
I think this change shouldn't be included in this PR

Reply:
Well now that I think about it again -> here we are actually updating the UiAction properties but in the QML we are updating the menuItem property so it's not needed. Correct?